sf_examples/envpool_examples/atari_examples/atari/envpool_atari_params.py

```diff
@@ -0,0 +1,63 @@
+import argparse
```

Comment:
Can this be maybe just envpool_examples/atari ? I feel like the directory structure is way too deep

Also, is there a chance we can reuse some of the Atari stuff between the envpool and regular Atari version?
Things like default parameters seem to be mostly the same, as well as the list of envs and things like that.

Duplicating code increases maintenance burden - the more code you have the more things to refactor each time the API changes.